logproof/src/fields.rs

```diff
@@ -62,6 +67,11 @@ pub struct SealQ128_8192 {}
  *
  * SEAL uses Q =
  * 0xffffee001, 0xffffc4001, 0x1ffffe0001
+ *
+ * This can be derived by running
+ * `CoefficientModulus::bfv_default(4096, SecurityLevel::TC128)`
+ * or by running the underlying SEAL function
+ * `CoeffModulus::BFVDefault(4096, sec_level_type::tc128)`
```

Comment:
Ditto do we use 2 or 3 modulii?

Reply:
For 4096 the number specified here is from the key modulus.